frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -373,7 +388,12 @@ export default function AnalysisParamForm({
             </WALabel>
             {checkedAdvanced || <WAOverlay />}
             <TwoParamContainer>
-              <Label htmlFor="prominenceFactorPeaks">Prominence(uN):</Label>
+              <Label
+                htmlFor="prominenceFactorPeaks"
+                title="Specifies how much a peak should stand out compared to the rest of the signal immediately surrounding it."
```

Comment:
We might want to more closely paraphrase the actual definition provided by scipy: "The prominence of a peak measures how much a peak stands out from the surrounding baseline of the signal and is defined as the vertical distance between the peak and its lowest contour line". We could do something like "Specifies the minimum required vertical distance between a local max and its lowest contour line to be classified as a peak"